charts/osm/templates/mutatingwebhook.yaml

```diff
@@ -27,4 +27,13 @@ webhooks:
       - key: "name"
         operator: NotIn
         values:
-        - {{.Release.Namespace}}
\ No newline at end of file
+        - {{.Release.Namespace}}
+  rules:
+    - apiGroups:
+        - ""
+      apiVersions:
+        - v1
+      operations:
+        - CREATE
+      resources:
+        - pods
```

Comment:
Do we still need to patch the rules in the mutatingwebhookconfiguration object here? https://github.com/openservicemesh/osm/blob/1646fb5b9e0578ded194152070535302b7a65aec/pkg/injector/webhook.go#L336

Can we have it in the template but not in webhook.go?